web-frontend/src/layouts/Placeholder/Placeholder.tsx

```diff
@@ -0,0 +1,22 @@
+import { FiList } from 'react-icons/fi';
+import Intl from '../../i18n/i18n';
+
+function Placeholder({ ...props }) {
+  const { message } = props;
+  return (
+    <div
+      className="flex flex-col justify-around items-center  
+        h-96 w-[700px] p-10 gap-10
+        relative left-20
```

Comment:
`left-20` ? 👀

Reply:
yeaaahhh it's hard to explain, but the walletLayout is not exactly centered, so to compensate you move to the left by the 20px

Reply:
centering it would ask for a rework of the layout and the tabs component, so I chose the most efficient route

Reply:
without left-10 :/

Reply:
Ok no problem!

Reply:
You should be able to centralize this content not using that...

Reply:
![Screenshot from 2023-06-06 17-48-44](https://github.com/massalabs/thyra-plugin-wallet/assets/90157528/4d877303-68d5-4d81-8167-ee5521cb6f47)

 this fixes the prob but it creates new ones 

![Screenshot from 2023-06-06 17-49-05](https://github.com/massalabs/thyra-plugin-wallet/assets/90157528/fae059bb-2749-4282-8503-7bdcbfca943c)

![Screenshot from 2023-06-06 17-49-10](https://github.com/massalabs/thyra-plugin-wallet/assets/90157528/ecd9aa78-a478-469a-aa21-a806eca582bb)

Reply:
> ![Screenshot from 2023-06-06 17-48-44](https://user-images.githubusercontent.com/90157528/243753536-4d877303-68d5-4d81-8167-ee5521cb6f47.png)
> 
> this fixes the prob but it creates new ones so would have to go into ui-kit and make changes
> 
> ![Screenshot from 2023-06-06 17-49-05](https://user-images.githubusercontent.com/90157528/243753646-fae059bb-2749-4282-8503-7bdcbfca943c.png)
> 
> ![Screenshot from 2023-06-06 17-49-10](https://user-images.githubusercontent.com/90157528/243753673-ecd9aa78-a478-469a-aa21-a806eca582bb.png)

For the `button`, we need to open an issue on the UI-Kit to fix this bug.

Reply:
To solve this problem.

Ref: https://github.com/massalabs/thyra-plugin-wallet/issues/345